### PR TITLE
stop tracking of .idea folder or directory

### PR DESCRIPTION
**Pull request by PuneetDabral**
## Fixes

stop tracking of .idea folder or directory #222

## Description

stop tracking of .idea folder

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by PuneetDabral** [comment]
sir check this..... now i stop tracking of .idea folder

**Comment by PuneetDabral** [comment]
using this command (git rm -r --cached path_to_your_folder/)

File: .gitignore [hunk 1]
```diff
@@ -2,7 +2,7 @@ venv
 es-venv
 
 # IDE junk
-.idea
+.idea/dictionaries
```
Comment (obulat) [inline .gitignore:5]: The value `.idea` here was correct: it tells git to ignore all the files and folders inside `.idea` folder.
If we change it to `.idea/dictionaries`, then only `dictionaries` will be ignored, and all the other contents of `.idea` folder will be tracked by git. Please, change it back to `.idea`.

`.idea/dictionaries` directory was added to the repository before `.gitignore` was created. So, git did not remove the dictionaries, even thow they were supposed to be ignored.

**Review by obulat — CHANGES_REQUESTED** [review]

**Comment by obulat** [comment]
Hi, @PuneetDabral, thank you for your contribution! The removal of dictionaries from the repository works great! 🎉 

I have added a small request for change inline. Also, please, add the required PR labels on the right side (we automate the checks for one of the `aspect:` and one of the `goal:` labels) and edit the PR message filling in all the relevant items, and removing irrelevant (for example, the `Screenshots` section).

**Comment by PuneetDabral** [comment]
can i again pr?

**Comment by PuneetDabral** [comment]
ok sir i will update it on 12 pm

**Comment by PuneetDabral** [comment]
check this sir i will update this...

**Comment by obulat** [comment]
> check this sir i was already update this...

There is still a change request that needs to be addressed (regarding new lines). 

Please also edit the PR message filling in all the relevant items, including the issue that this PR fixes, checking the relevant checklist items, and removing irrelevant sections(for example, the Screenshots section).

**Comment by PuneetDabral** [comment]
so now what i can change ??

**Comment by obulat** [comment]
> so now what i can change ??

Click on 'Edit' in your first message:
PRMessage
And add the missing details. You can see an example of a PR message in https://github.com/WordPress/openverse-api/pull/224

Also, address this request: https://github.com/WordPress/openverse-api/pull/222/files#r713258290 . If you do not delete the extra lines, linting will fail, and we will not be able to merge the PR.

**Comment by PuneetDabral** [comment]
then

**Review by dhruvkb — APPROVED** [review]
This should effectively stop tracking the `.idea/` directory.

**Review by obulat — APPROVED** [review]

**Comment by obulat** [comment]
Thank you for your contribution, @PuneetDabral !

**Pull request merged**